internal/api/v3/handler_payment_initiations_create.go

```diff
@@ -115,13 +115,18 @@ func paymentInitiationsCreate(backend backend.Backend) http.HandlerFunc {
 			pi.DestinationAccountID = pointer.For(models.MustAccountIDFromString(*payload.DestinationAccountID))
 		}
 
-		err = backend.PaymentInitiationsCreate(ctx, pi, noValidation)
+		task, err := backend.PaymentInitiationsCreate(ctx, pi, noValidation, false)
 		if err != nil {
 			otel.RecordError(span, err)
 			handleServiceErrors(w, r, err)
 			return
 		}
 
+		if noValidation {
+			api.Accepted(w, task)
+			return
+		}
+
```

Comment:
What is the purpose of spliting the possible response values based on a flag in v3?

Reply:
You're right, let's not split the possible response status code